os/compression/compress_read.c

```diff
@@ -108,31 +108,47 @@ static void compress_blocks_to_read(int *first_block, int *last_block, int *no_b
  *
  * Returned value:
  *   OK (0) is Success
- *   ERROR (-1) on Failure
+ *   Negative value on Failure
  ****************************************************************************/
 static int compress_parse_header(int filfd, uint16_t offset)
 {
 	off_t rpos;					/* Position returned by lseek */
 	int nbytes;					/* Number of bytes read  */
+	int compheader_size;				/* Total size of compression header */
 
-	/* Seek to the next read position */
-
+	/* Seek to location of size of compression header */
 	rpos = lseek(filfd, offset, SEEK_SET);
 	if (rpos != offset) {
 		int errval = get_errno();
 		bmdbg("ERROR : leek to offset %lu failed: %d\n", (unsigned long)offset, errval);
```

Comment:
Typing mistake : "leek" written instead of "seek"

Reply:
Done